### PR TITLE
Maps.week4

### PR DESCRIPTION
**Pull request by bosticc**
This is for the maps api of week 4.

### CODE REVIEW

**Review by askewc — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/metMylesServlet.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+@WebServlet("/met-Myles")
+public class metMylesServlet extends HttpServlet {
+
+  public Map<String, Integer> metVotes = new HashMap<>();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Votes").addSort("timestamp", SortDirection.DESCENDING);
+
+    //Makes a new datastore object and intializes it with the comment
```
Comment (zkoogle) [inline portfolio/src/main/java/com/google/sps/servlets/metMylesServlet.java:44]: Can you change all the comment style consistent:
// Add a space after the slashes and start with a capital letter.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,17 +12,110 @@
...
+                   'height':300};
+
+    // Instantiate and draw our chart, passing in some options.
+    var foodChart = new google.visualization.PieChart(document.getElementById('chart_div'));
+    foodChart.draw(dataPie, options);
+
+    const dataMyles = new google.visualization.DataTable();
+    //fetch data from servlet for bar chart 
+    fetch('/met-Myles').then(response => response.json())
+    .then((metVotes) => {
```
Comment (zkoogle) [inline portfolio/src/main/webapp/script.js:78]: Proper indentation please.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,110 @@
...
+  });
+}
+ 
+//Prints out the number of comments that the user specifies
+function numComments(commentsNum) {
+  fetch('/data?viewComments=' + commentsNum).then(response => response.json()).then((comments) => {
+    const taskListElement = document.getElementById('task-list');
+    taskListElement.innerHTML="";
+    for (let i = 0; i < commentsNum; i++)
+    {
```
Comment (zkoogle) [inline portfolio/src/main/webapp/script.js]: Move "{" to the above line.

**Comment by askewc** [comment]
Huzzah! Looks like you can merge this puppy! 

yuss

**Comment by bosticc** [comment]
> Huzzah! Looks like you can merge this puppy!
> 
> yuss

::::))))))

**Pull request merged**